pkg/kubectl/cmd/get/get.go

```diff
@@ -210,6 +210,12 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 	if err != nil {
 		return err
 	}
+	// For kubectl get events Added default sorting by lastTimestamp
+	if sortBy == "" {
+		if len(args) == 1 && args[0] == "events" {
```

Comment:
Rather than special casing, we'd probably want to do something like create a map of resource type -> default sort order

Reply:
I have modified the code as per the above comments.